src/main/java/model/Monster.java

```diff
@@ -79,11 +79,8 @@ public void move() {
                     spriteBase.getX(), spriteBase.getY(), newX, newY));
         }
 
-        checkPowerups();
-        if (this.isReducedSpeed) {
-            setSpeed(Settings.MONSTER_SLOWDOWN_FACTOR * Settings.MONSTER_SPEED);
-        }
-
+        this.setChanged();
```

Comment:
This already happens in the AnimationTimer and should be removed.

<organisation